_data/students.yml

```diff
@@ -108,3 +108,10 @@
   facebook: https://www.facebook.com/saicharan.reddy.927
   bio: New to github!
 
+
+- name: Pradeep Gangwar
```

Comment:
I don't know what I'm doing wrong. I'm following the guidelines given on github help pages.. Doesn't seem to work...